pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -60,6 +60,27 @@ var userRegistriesFile = filepath.FromSlash(".config/containers/registries.conf"
 // userRegistriesDir is the path to the per user registry configuration file.
 var userRegistriesDir = filepath.FromSlash(".config/containers/registries.conf.d")
 
+// SetRegistriesConfPath sets the SystemRegistriesConfPath in the system
+// context, unless already set.  Possible values are, in priority and only if
+// set, the CONTAINERS_REGISTRIES_CONF or REGISTRIES_CONFIG_PATH environment
+// variable.
+func SetRegistriesConfPath(sys *types.SystemContext) {
```

Comment:
`error` return to support (optional, future, not currently implemented) validation like pointing at nonexistent or inaccessible paths?